CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## [0.26.0](https://github.com/eclipse/lemminx/milestone/42?closed=1) (June 15, 2023)
+
+### Enhancements
+
+ * TextDocument to expose an API for getting a document line number by offset. See [#1519](https://github.com/eclipse/lemminx/pull/1519).
+ * Make ICodeActionParticipant members default. See [#1518](https://github.com/eclipse/lemminx/issues/1518).
+ * Allow code actions to be created if no Diagnostics is provided. See [#1516](https://github.com/eclipse/lemminx/issues/1516).
+ * IRenameParticipant.doRename() is not conform with LSP-defined rename result code action. See [#1521](https://github.com/eclipse/lemminx/issues/1521).
+
+### Bug Fixes
+
+ * xml.foldings.includeClosingTagInFold has no effect in binary mode. See [#1523](https://github.com/eclipse/lemminx/issues/1523).
```

Comment:
I would just add back-ticks to `xml.foldings.includeClosingTagInFold`

Reply:
Ack